packages/domain-picker/src/styles/placeholder.scss

```diff
@@ -0,0 +1,29 @@
+@import './base-styles.scss';
+
+// ==========================================================================
+// Placeholder mixin
+// Adds animation to placeholder section
+// ==========================================================================
+
+@mixin placeholder( $color-property: $light-gray-200 ) {
+	animation: loading-fade 1.6s ease-in-out infinite;
+	background: $color-property;
+	color: transparent;
+
+	&::after {
+		content: '\00a0';
+	}
+}
+
+// Another pulsing animation for placeholders
+@keyframes loading-fade {
```

Comment:
Let's prefix this with something because keyframes end up in global scope and end up colliding with other with same name.